modules/ztp-acm-installing-disconnected-rhacm.adoc

```diff
@@ -12,6 +12,12 @@ You use {rh-rhacm-first} on a hub cluster in the disconnected environment to man
 * Install the {product-title} CLI (`oc`).
 * Log in as a user with `cluster-admin` privileges.
 * Configure a disconnected mirror registry for use in the cluster.
++
+[NOTE]
+====
+If you want to deploy Operators to the spoke clusters, you must also add them to this registry.
```

Comment:
We capitalize "operator" in a general context like this? (We don't appear to do the same for other terms, such as "custom resource.")

Reply:
Capitalizing Operator is an OpenShift convention. Generally, capitalization follows our style guidelines, where very few terms are capitalized, unless they are proper product names.